README.md

```diff
@@ -181,6 +182,10 @@ For Windows, [we release images](#windows-images) only for amd64 CPU architectur
 - Windows Server 2019
 - Windows Server 2022
 
+#### Debug images
+
+Deploying AWS for Fluent `debug` images can help the AWS team troubleshoot an issue. If you experience a bug, especially a [crash/SIGSEGV issue](https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#segfaults-and-crashes-sigsegv), then please consider deploying the debug version of the image. After a crash, the debug image can print out a stacktrace and upload a core dump to S3. See our [debugging guide](https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#guide-to-debugging-fluent-bit-issues) for more info on using debug images.
```

Comment:
1st link should go here: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#caught-signal-sigsegv

2nd should go here: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#1-build-and-distribute-a-core-dump-s3-uploader-image

Reply:
Please also give the tag format, if I want debug image for 2.X.Y, what tag is it?